wiki/howToUseSemantics.md

```diff
@@ -0,0 +1,76 @@
+<div id="main_content_wrap" class="outer">
+
+<section id="main_content" class="inner">
+
+## How to use semantics
+
+To import the project user need to add to pom.xml
+
+<div>
+
+    <repositories>
+       <repository>
+           <id>jitpack.io</id>
+           <url>https://jitpack.io</url>
+       </repository>
+    </repositories>
+    <dependency>
+        <groupId>com.github.Ericsson</groupId>
+        <artifactId>eiffel-remrem-semantics</artifactId>
+        <version>x.x.x</version>
+    </dependency>
+
+   </div>
+
+Or import the project to pom.xml  
+
+<div>
+
+    repositories {
+         ...
+         maven { url 'https://jitpack.io' }
+    }
+    dependencies {
+        compile("com.github.Ericsson:eiffel-remrem-semantics:x.x.x")
```

Comment:
again with the capital E. If it's not case sensitive I suggest changing it to lowercase.

Reply:
changed com.github.Ericsson to com.github.eiffel-community